packages/core/admin/ee/admin/src/pages/SettingsPage/pages/ReviewWorkflows/utils/colors.ts

```diff
@@ -33,7 +34,7 @@ export function getStageColorByHex(hex) {
 
 export function getAvailableStageColors() {
   return Object.entries(STAGE_COLORS).map(([themeColorName, name]) => ({
-    hex: lightTheme.colors[themeColorName].toUpperCase(),
+    hex: lightTheme.colors[themeColorName as keyof typeof lightTheme.colors].toUpperCase(),
```

Comment:
```suggestion
    hex: lightTheme.colors[themeColorName as keyof DefaultTheme['colors']].toUpperCase(),
```
That should work and avoid the typeof call?